matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -179,6 +183,36 @@ var PhenoTips = (function (PhenoTips) {
         this._currentSortingOrder = "none";
     },
 
+    _initiateDisclaimers: function() {
+        $$('#checkbox-server-filters .fa').each(function(trigger) {
+            var disclaimerTextInput = trigger.up('label').down('.disclaimer');
+            if (disclaimerTextInput && disclaimerTextInput.value) {
+
+                var disclaimerContainer = new Element('div');
+                disclaimerContainer.insert(new Element('p', {'class' : 'mme-disclaimer'}).insert(disclaimerTextInput.value));
+
+                var endTimeInput = trigger.up('label').down('.endTime');
+                var lastUpdateText = this._MATCHES_NEVER_RUN;
+                if (endTimeInput && endTimeInput.value) {
+                	var time = new Date(endTimeInput.value);
+                    lastUpdateText = this._MATCHES_LAST_RUN + " " + time.toISOString().split('T')[0];
+                }
+                disclaimerContainer.insert(new Element('div').insert(lastUpdateText));
```

Comment:
This should be colored as specified in the Jira issue to make it stand out more.